enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -0,0 +1,353 @@
+---
+title: mixed-cpu-node-plugin
+authors:
+  - "@Tal-or"
+reviewers: 
+  - "@MarSik"
+  - "@yanirq"
+  - "@ffromani"
+  - "@jmencak"
+  - "@mrunalp"
+  - "@dhellmann"
+approvers: 
+  - "@mrunalp"
+  - "@dhellmann"
+api-approvers:
+  - "@mrunalp"
+  - "@dhellmann"
+creation-date: 2023-05-03
+last-updated: 2023-06-14
+tracking-link: 
+  - https://issues.redhat.com/browse/CNF-7603
+  - https://github.com/openshift/enhancements/pull/1421
```

Comment:
Does this work depend on the shared & guaranteed CPU pools design in this enhancement?

Reply:
Not depends but related. In the future we'll use the same API for specifying the `shared-cpus`

Reply:
UPDATE: mixed-cpu uses the same `spec.cpu.shared` field to represents the shared cpus since day one.
Initially we thought to use an annotation first but reviewers didn't like it so we changed this decision.
EP updated accordingly.